opentelemetry-sdk/tests/trace/export/test_export.py

```diff
@@ -21,9 +21,14 @@
 from unittest import mock
 
 from opentelemetry import trace as trace_api
-from opentelemetry.configuration import Configuration
 from opentelemetry.context import Context
 from opentelemetry.sdk import trace
+from opentelemetry.sdk.environment_variables import (
+    OTEL_BSP_EXPORT_TIMEOUT,
+    OTEL_BSP_MAX_EXPORT_BATCH_SIZE,
+    OTEL_BSP_MAX_QUEUE_SIZE,
+    OTEL_BSP_SCHEDULE_DELAY,
```

Comment:
Nice, didn't know you removed the _MILLIS in this PR itself. That was updated very recently in the spec. I should close https://github.com/open-telemetry/opentelemetry-python/issues/1554